redfish-core/schema/oem/openbmc/json-schema/OpenBMCManager.v1_0_0.json

```diff
@@ -256,55 +298,16 @@
             "properties": {},
             "type": "object"
         },
-        "Oem": {
```

Comment:
Shouldn't these go in an IBMManager?

Reply:
OpenBMCManager.json is the same as upstream OpenBMCManager.json - https://github.com/openbmc/bmcweb/blob/6be832e2963e9d720dd95543358eca380c5e52d2/redfish-core/schema/oem/openbmc/json-schema/OpenBMCManager.v1_0_0.json#L2

IBM part is moved out of OpenBMC schem to IBM schema namespace.